cmd/kubelet/app/plugins.go

```diff
@@ -76,6 +77,7 @@ func ProbeNetworkPlugins(pluginDir string) []network.NetworkPlugin {
 
 	// for each existing plugin, add to the list
 	allPlugins = append(allPlugins, exec.ProbeNetworkPlugins(pluginDir)...)
+	allPlugins = append(allPlugins, cni.ProbeNetworkPlugins(pluginDir)...)
```

Comment:
oh I see what you did - repurposing the existing plugin dir flag... need to think about that